spec/queries/find_exact_query_spec.rb

```diff
@@ -169,79 +173,79 @@
-        topics: { any: %w[uuid-888] },
+        browse_pages: { any: %w[uuid-888] },
         policies: { any: %w[pol-123 pol-555] },
       })
       query = build_query(tags: {
-        topics: { any: %w[uuid-999] },
+        browse_pages: { any: %w[uuid-999] },
         policies: { any: %w[pol-456 pol-666] },
       })
       expect(query.exact_match).to be_nil
     end
 
     it "matched when matching keys with matching values" do
       subscriber_list = create_subscriber_list(tags: {
-        topics: { any: %w[uuid-888] },
+        browse_pages: { any: %w[uuid-888] },
         policies: { any: %w[pol-123 pol-555] },
       })
       query = build_query(tags: {
-        topics: { any: %w[uuid-888] },
+        browse_pages: { any: %w[uuid-888] },
         policies: { any: %w[pol-123 pol-555] },
       })
       expect(query.exact_match).to eq(subscriber_list)
     end
 
     it "order of values for keys does not affect matching" do
       subscriber_list = create_subscriber_list(tags: {
-        topics: { any: %w[uuid-888] },
+        browse_pages: { any: %w[uuid-888] },
         policies: { any: %w[pol-123 pol-555] },
       })
       query = build_query(tags: {
         policies: { any: %w[pol-555 pol-123] },
-        topics: { any: %w[uuid-888] },
+        browse_pages: { any: %w[uuid-888] },
       })
       expect(query.exact_match).to eq(subscriber_list)
     end
 
     it "requires and and any operators to be correctly set" do
       subscriber_list = create_subscriber_list(tags: {
-        topics: { all: %w[uuid-888 uuid-999], any: %w[uuid-777] },
+        browse_pages: { all: %w[uuid-888 uuid-999], any: %w[uuid-777] },
         subject: { all: %w[subject-123 subject-555] },
       })
       query = build_query(tags: {
         subject: { all: %w[subject-123 subject-555] },
-        topics: { all: %w[uuid-888 uuid-999], any: %w[uuid-777] },
+        browse_pages: { all: %w[uuid-888 uuid-999], any: %w[uuid-777] },
       })
       expect(query.exact_match).to eq(subscriber_list)
     end
 
     it "requires both operators to be present" do
       subscriber_list = create_subscriber_list(tags: {
-        topics: { all: %w[uuid-888], any: %w[uuid-777] },
+        browse_pages: { all: %w[uuid-888], any: %w[uuid-777] },
       })
-      bad_query = build_query(tags: { topics: { all: %w[uuid-888] } })
-      good_query = build_query(tags: { topics: { all: %w[uuid-888], any: %w[uuid-777] } })
+      bad_query = build_query(tags: { browse_pages: { all: %w[uuid-888] } })
+      good_query = build_query(tags: { browse_pages: { all: %w[uuid-888], any: %w[uuid-777] } })
       expect(bad_query.exact_match).to be_nil
       expect(good_query.exact_match).to eq(subscriber_list)
     end
 
     it "matched when subscriber tags has the same tags" do
       query = build_query(tags: { policies: { any: %w[beer] },
-                                  topics: { all: %w[taxon] } })
+                                  browse_pages: { all: %w[taxon] } })
```

Comment:
`browse_pages` aren't a valid tag - let's update this to be `taxons` as well.

Reply:
Please, check the comment above.

Reply:
Yes, that makes sense. Specialist publisher bypasses publishing-api, so its tags are not there. I'll use some of those instead of policies.